extensions/amp-story/0.1/test/test-amp-story.js

```diff
@@ -13,12 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {AnalyticsTrigger} from '../analytics';
 import {AmpStory} from '../amp-story';
+import {AmpStoryPage} from '../amp-story-page';
```

Comment:
This works without the file-level import? `import '../amp-story'`?

Reply:
Yes, the amp-story-page is properly instantiated